src/egon/data/datasets/scenario_parameters/parameters.py

```diff
@@ -47,6 +90,100 @@ def electricity(scenario):
+                "s_nom": 260,  # [MVA]
+                "R": 0.109,  # [Ohm/km]
+                "L": 1.2,  # [mH/km]
+            },
+            "ac_cable_110kV": {
+                "s_nom": 280,  # [MVA]
+                "R": 0.0177,  # [Ohm/km]
+                "L": 0.3,  # [mH/km]
+            },
+            "ac_line_220kV": {
+                "s_nom": 520,  # [MVA]
+                "R": 0.109,  # [Ohm/km]
+                "L": 1.0,  # [mH/km]
+            },
+            "ac_cable_220kV": {
+                "s_nom": 550,  # [MVA]
+                "R": 0.0176,  # [Ohm/km]
+                "L": 0.3,  # [mH/km]
+            },
+            "ac_line_380kV": {
+                "s_nom": 1790,  # [MVA]
+                "R": 0.028,  # [Ohm/km]
+                "L": 0.8,  # [mH/km]
+            },
+            "ac_cable_380kV": {
+                "s_nom": 925,  # [MVA]
+                "R": 0.0175,  # [Ohm/km]
+                "L": 0.3,  # [mH/km]
+            },
+        }
+
+        # Insert capital costs
+        # Source for grid costs: Netzentwicklungsplan Strom 2035, Version 2021, 2. Entwurf
+        parameters["capital_cost"] = {
+            "ac_ehv_overhead_line": 2.5e6
+            / parameters["electrical_parameters"]["ac_line_380kV"][
+                "s_nom"
+            ],  # [EUR/km/MW]
+            "ac_ehv_cable": 11.5e6
+            / parameters["electrical_parameters"]["ac_cable_380kV"][
+                "s_nom"
+            ],  # [EUR/km/MW]
+            "dc_overhead_line": 0.5e3,  # [EUR/km/MW]
+            "dc_cable": 3.25e3,  # [EUR/km/MW]
+            "dc_inverter": 0.3e6,  # [EUR/MW]
+            "transformer_380_110": 17.33e3,  # [EUR/MVA]
+            "transformer_380_220": 13.33e3,  # [EUR/MVA]
+            "transformer_220_110": 17.5e3,  # [EUR/MVA]
+            "battery": read_costs(2035, "battery inverter", "investment")
+            + parameters["efficiency"]["battery"]["max_hours"]
+            * read_costs(2035, "battery storage", "investment"),  # [EUR/MW]
+        }
+
+        # Insert marginal_costs in EUR/MWh
+        # marginal cost can include fuel, C02 and operation and maintenance costs
+        parameters["marginal_cost"] = {
+            "oil": global_settings(scenario)["fuel_costs"]["oil"]
+            + read_costs(2035, "oil", "VOM")
+            + global_settings(scenario)["co2_costs"]
+            * global_settings(scenario)["co2_emissions"]["oil"],
+            "other_non_renewable": global_settings(scenario)["fuel_costs"][
+                "gas"
+            ]
+            + global_settings(scenario)["co2_costs"]
+            * global_settings(scenario)["co2_emissions"][
+                "other_non_renewable"
+            ],
+            "wind_offshore": read_costs(2035, "offwind", "VOM"),
+            "wind_onshore": read_costs(2035, "onwind", "VOM"),
+            "pv": read_costs(2035, "solar", "VOM"),
+            "OCGT": read_costs(2035, "OCGT", "VOM"),
+        }
+
```

Comment:
Reading the .csv file is comparably computation heavy in pandas, we should consider reading the cost table from csv once, I think.